doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -16,13 +16,8 @@ accredited testing laboratory.
 The implementations in this provider specifically have these properties
 defined:
 
-=over 4
-
-"provider=default"
-
-"fips=yes"
-
-=back
+    provider=default
+    fips=yes
```

Comment:
Drop this change, @slontis has a better fix in #11610.  If you need to, wait for it to be merged (should happen soon)

Reply:
ok